DataFormats/GEMDigi/interface/GEMPadDigi.h

```diff
@@ -33,12 +37,18 @@ class GEMPadDigi {
   int16_t bx() const { return bx_; }
   GEMSubDetId::Station station() const { return station_; }
 
+  // Newer GE2/1 geometries will have 16! eta partitions
+  // instead of the usual 8.
+  void setNPartitions(unsigned nPart) { part_ = nPart; }
```

Comment:
will there ever be a case where the `part_` value needs to be changed after the digi object is created?

Reply:
Very unlikely. I added the function for completeness.

Reply:
There aren't setters for the other attributes... it might be safer to remove it.